File: 가운데 글자 가져오기.c
#include <stdio.h>
#include <stdbool.h>
#include <stdlib.h>

char* solution(const char* s) {
    char* answer = (char*)malloc(1);
    int k = strlen(s);
    
    char arr[k];
    char arrs[3];
    
    strcpy(arr, s);
    
    if (k % 2 == 0){
        k = k/2;
        arrs[0] = arr[k-1];
        arrs[1] = arr[k];
    }
    else{
        k = k/2;
        arrs[0] = arr[k];
    }
    
    strcpy(answer, arrs);
    return answer;
}

File: 가운데 글자 가져오기.py
"""
def solution(s):
    if len(s) % 2 == 1:
        return s[(len(s)//2)]
    else: 
        return s[(len(s)//2)-1:(len(s)//2)+1]
"""
#2022.11.13
def solution(s):
    if len(s) % 2 == 0:
        return s[(len(s)//2)-1:(len(s)//2)+1]
    return s[len(s)//2]
